api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -208,79 +210,57 @@ default void listRuntimeAfterMapper(
-  @Mapping(target = "appType", ignore = true)
-  UserAppEnvironment toApiApp(LeonardoGetAppResponse app);
-
-  @Mapping(target = "createdDate", source = "auditInfo.createdDate")
-  @Mapping(target = "dateAccessed", source = "auditInfo.dateAccessed")
-  @Mapping(target = "creator", source = "auditInfo.creator")
-  @Mapping(target = "autopauseThreshold", ignore = true)
-  @Mapping(
-      target = "googleProject",
-      source = "cloudContext",
-      qualifiedByName = "legacy_cloudContextToGoogleProject")
-  @Mapping(target = "appType", ignore = true)
-  UserAppEnvironment toApiApp(LeonardoListAppResponse app);
+  UserAppEnvironment toApiApp(ListAppResponse app);
 
   KubernetesRuntimeConfig toKubernetesRuntimeConfig(
-      LeonardoKubernetesRuntimeConfig leonardoKubernetesRuntimeConfig);
+      org.broadinstitute.dsde.workbench.client.leonardo.model.KubernetesRuntimeConfig
+          leonardoKubernetesRuntimeConfig);
 
-  LeonardoKubernetesRuntimeConfig toLeonardoKubernetesRuntimeConfig(
-      KubernetesRuntimeConfig kubernetesRuntimeConfig);
+  @Mapping(target = "additionalProperties", ignore = true)
+  org.broadinstitute.dsde.workbench.client.leonardo.model.KubernetesRuntimeConfig
+      toLeonardoKubernetesRuntimeConfig(KubernetesRuntimeConfig kubernetesRuntimeConfig);
 
   // SAS and RStudio apps are implemented as ALLOWED Helm Charts
   @ValueMapping(source = "RSTUDIO", target = "ALLOWED")
   @ValueMapping(source = "SAS", target = "ALLOWED")
-  LeonardoAppType toLeonardoAppType(AppType appType);
+  org.broadinstitute.dsde.workbench.client.leonardo.model.AppType toLeonardoAppType(
+      AppType appType);
 
   // Cromwell is not an ALLOWED Helm Chart in Leonardo
   @ValueMapping(source = "CROMWELL", target = MappingConstants.NULL)
-  LeonardoAllowedChartName toLeonardoAllowedChartName(AppType appType);
+  AllowedChartName toLeonardoAllowedChartName(AppType appType);
 
   @ValueMapping(source = "BALANCED", target = MappingConstants.NULL)
   DiskType toDiskType(org.broadinstitute.dsde.workbench.client.leonardo.model.DiskType diskType);
 
-  @AfterMapping
-  default void listAppsAfterMapper(
-      @MappingTarget UserAppEnvironment appEnvironment, LeonardoListAppResponse listAppResponse) {
-    setAppType(appEnvironment, listAppResponse.getLabels());
-  }
-
-  @AfterMapping
-  default void getAppAfterMapper(
-      @MappingTarget UserAppEnvironment appEnvironment, LeonardoGetAppResponse getAppResponse) {
-    setAppType(appEnvironment, getAppResponse.getLabels());
-  }
-
-  default void setAppType(UserAppEnvironment appEnvironment, @Nullable Object appLabels) {
-    appEnvironment.setAppType(
-        LeonardoLabelHelper.maybeMapLeonardoLabelsToGkeApp(appLabels)
-            .orElseThrow(
-                () ->
-                    new ServerErrorException(
-                        String.format(
-                            "Missing app type labels for app with labels %s", appLabels))));
+  @Named("mapAppType")
```

Comment:
I don't see this map being used before. What is this for?

Reply:
This was part of `listAppsAfterMapper()` which was implicitly called as an `@AfterMapping` after the main `toApiApp(LeonardoListAppResponse app)` was called.

AfterMappings are a bit error-prone to work with, so it's better to make the association explicit where possible.  This new `mapAppType()` is that more explicit version.